.env.development

```diff
@@ -28,3 +28,4 @@ LEARNER_RECORD_MFE_BASE_URL='http://localhost:1990'
 COLLECT_YEAR_OF_BIRTH=true
 APP_ID=''
 MFE_CONFIG_API_URL=''
+ENABLE_SKILLS_BUILDER='true'
```

Comment:
Should we add this to the base `.env` file as well?

Assumption: We want the base `.env` file to have a list/default of _all_ available settings to the app. The `env.development` file contains smart overrides of the base file options to facilitate development.